src/components/tools/geometry-tool/geometry-content.tsx

```diff
@@ -557,6 +559,26 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
     }
   }
 
+  private handleArrowKeys = (e: React.KeyboardEvent, keys: string) => {
+    const { board } = this.state;
+    const selectedObjects = board && this.getContent().selectedObjects(board);
+    const selectedPoints = selectedObjects && selectedObjects.filter(isPoint);
+    const hasSelectedPoints = selectedPoints ? selectedPoints.length > 0 : false;
+    let dx = 0;
+    let dy = 0;
+    switch (keys) {
+      case "left":  dx = -kSnapUnit; break;
+      case "right": dx =  kSnapUnit; break;
+      case "up":    dy =  kSnapUnit; break;
+      case "down":  dy = -kSnapUnit; break;
+    }
```

Comment:
Do we need to do any bounds checking here?  Is it safe to nudge a point outside of the visible graph area?  I suppose in the case where the point is part of a shape, the user can always delete the shape if they nudge a point offscreen (and hence it is no longer selectable).

Reply:
The story doesn't specify any and it's not obvious to me what the best response would be: stop moving at the edge of the visible axis bounds? auto-enlarge the axis bounds? Users can always manually increase the axis bounds if necessary. My inclination is to implement it this way and if it becomes a problem we can write another story.

Reply:
Agreed.  I think there are enough ways to recover if a point is nudged out of bounds.